tests/integration/test_image_filters.py

```diff
@@ -633,3 +633,823 @@ def test_hysteresis_threshold_raises_exception_on_thresholds(low, high):
 
     assert isinstance(err.value, Exception)
     assert str(err.value) == "thresholds cannot be None"
+
+
+@pytest.mark.parametrize(
+    "pil_image, expected_image, disk_size",
+    (
+        (
+            GS.DIAGNOSTIC_SLIDE_THUMB_GS,
+            "pil-images-gs/diagnostic-slide-thumb-gs1-local-otsu",
+            10,
+        ),
+        (
+            GS.DIAGNOSTIC_SLIDE_THUMB_GS,
+            "pil-images-gs/diagnostic-slide-thumb-gs2-local-otsu",
+            3.8,
+        ),
+        (
+            GS.DIAGNOSTIC_SLIDE_THUMB_GS,
+            "pil-images-gs/diagnostic-slide-thumb-gs3-local-otsu",
+            0,
+        ),
+        (
+            GS.DIAGNOSTIC_SLIDE_THUMB_GS,
+            "pil-images-gs/diagnostic-slide-thumb-gs4-local-otsu",
+            np.sqrt(2),
+        ),
```

Comment:
For my understanding : what about if the disk size is > than the input image? Is this case meaningful ? So we need a new test case with a big number as disk size?

Reply:
not meaningful, but does not raise an Exception